src/arviz_plots/plots/pairs_focus_plot.py

```diff
@@ -0,0 +1,222 @@
+
+from arviz_plots.plot_collection import PlotCollection
+from arviz_plots.plots.utils import (
+    filter_aes,
+    get_group,
+    process_group_variables_coords,
+    set_wrap_layout,
+)
+from arviz_plots.visuals import divergence_scatter, labelled_x, labelled_y, scatter_x
+
+
+def plot_pairs_focus(
+    dt,
+    var_names=None,
+    filter_vars=None,
+    group="posterior",
+    coords=None,
+    sample_dims=None,
+    focus_var=None,
+    focus_var_coords=None,
+    plot_collection=None,
+    backend=None,
+    labeller=None,
+    aes_by_visuals=None,
+    visuals=None,
+    **pc_kwargs,
+):
+    """Plot a fixed variable against other variables in the dataset.
+
+    Parameters
+    ----------
+    dt : DataTree
+        Input data
+    var_names: str or list of str, optional
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars: {None, “like”, “regex”}, optional, default=None
+        If None (default), interpret var_names as the real variables names.
+        If “like”, interpret var_names as substrings of the real variables names.
+        If “regex”, interpret var_names as regular expressions on the real variables names.
+    group : str, optional
+        Group to use for plotting. Defaults to "posterior".
+    coords : mapping, optional
+        Coordinates to use for plotting. Defaults to None.
+    sample_dims : iterable, optional
+        Dimensions to reduce unless mapped to an aesthetic.
+        Defaults to ``rcParams["data.sample_dims"]``
+    focus_var: str
+        Name of the variable to be plotted against all other variables.
+    focus_var_coords : mapping, optional
+        Coordinates to use for the target variable. Defaults to None.
+    plot_collection : PlotCollection, optional
+    backend : {"matplotlib", "bokeh","plotly"}, optional
+    labeller : labeller, optional
+    aes_by_visuals : mapping, optional
+        Mapping of visuals to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Valid keys are the same as for `visuals`.
+    visuals : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * scatter -> passed to :func:`~.visuals.scatter_x`
+        * divergence -> passed to :func:`~.visuals.divergence_scatter`. Defaults to False.
+        * title -> :func:`~.visuals.labelled_title`
+
+    pc_kwargs : mapping
+        Passed to :class:`arviz_plots.PlotCollection`
+
+    Returns
+    -------
+    PlotCollection
+
+    Examples
+    --------
+    Default plot_pair_focus
+
+    .. plot::
+        :context: close-figs
+
+        >>> from arviz_plots import plot_pairs_focus, style
+        >>> style.use("arviz-variat")
+        >>> from arviz_base import load_arviz_data
+        >>> dt = load_arviz_data('centered_eight')
+        >>> plot_pairs_focus(
+        >>>     dt,
+        >>>     var_names=["theta", "tau"],
+        >>>     focus_var="mu",
+        >>> )
```

Comment:
we have the exact same example in the example gallery, so it should not be repeated here too. We could for example show how to use `focus_coords`